### PR TITLE
fix blank lines being passed to parse message

### PR DESCRIPTION
**Pull request by helderroem**
I was using this to interact with twitch irc and it kept crashing because the service occasionally sends a double line break at the end of a line, this was an easy fix but I'm not sure it's in the right place. Feel free to refuse it.

### CODE REVIEW

File: lib/irc.js [hunk 1]
```diff
@@ -749,13 +749,15 @@ Client.prototype.connect = function(retryCount, callback) {
...
 
         lines.forEach(function iterator(line) {
-            var message = parseMessage(line, self.opt.stripColors);
-
-            try {
-                self.emit('raw', message);
-            } catch (err) {
-                if (!self.conn.requestedDisconnect) {
-                    throw err;
+            if (line.length > 0) {
```
Comment (jirwin) [inline lib/irc.js]: Probably best just to check the truthyness of `line`. The empty string will evaluate as false.

**Comment by jirwin** [comment]
Looks pretty good to me. Thanks!

**Comment by helderroem** [comment]
You're right, updated it to just be a truth check. and thanks you this module works great!

**Comment by ChadKeating** [comment]
Thank you for this fix.

**Comment by apeiron** [comment]
Why does this change the iconv implementation used? Can you add tests, and squash your commits into one as described in the [contributing guide](https://github.com/martynsmith/node-irc/blob/master/CONTRIBUTING.md)?

**Comment by helderroem** [comment]
Sorry, continued working on the wrong branch. I've removed iconv-lite, added a test and squashed commits.

**Comment by jirwin** [comment]
+1

**Pull request merged**